packages/account/src/BiconomySmartAccountV2.ts

```diff
@@ -94,127 +129,55 @@ export class BiconomySmartAccountV2 extends BaseSmartAccount {
    */
   public static async create(biconomySmartAccountConfig: BiconomySmartAccountV2Config): Promise<BiconomySmartAccountV2> {
     const instance = new BiconomySmartAccountV2(biconomySmartAccountConfig);
-    instance.factoryAddress = biconomySmartAccountConfig.factoryAddress ?? DEFAULT_BICONOMY_FACTORY_ADDRESS; // This would be fetched from V2
-
-    if (biconomySmartAccountConfig.biconomyPaymasterApiKey) {
-      instance.paymaster = new BiconomyPaymaster({
-        paymasterUrl: `https://paymaster.biconomy.io/api/v1/${biconomySmartAccountConfig.chainId}/${biconomySmartAccountConfig.biconomyPaymasterApiKey}`,
-      });
-    }
-
-    const defaultFallbackHandlerAddress =
-      instance.factoryAddress === DEFAULT_BICONOMY_FACTORY_ADDRESS
-        ? DEFAULT_FALLBACK_HANDLER_ADDRESS
-        : biconomySmartAccountConfig.defaultFallbackHandler;
-    if (!defaultFallbackHandlerAddress) {
-      throw new Error("Default Fallback Handler address is not provided");
-    }
-    instance.accountAddress = biconomySmartAccountConfig.senderAddress ?? undefined;
```

Comment:
we still need these in init right?

Reply:
Instead of doing these in the init/create have moved it to private constructor.
Keeping the `create` method to do async things. So I call constructor where I do all this and then call async tasks like this

```ts
if (biconomySmartAccountConfig.defaultValidationModule) {
      instance.defaultValidationModule = biconomySmartAccountConfig.defaultValidationModule;
    } else {
      instance.defaultValidationModule = await ECDSAOwnershipValidationModule.create({
        signer: biconomySmartAccountConfig.signer!,
    });
}
```